source/pmc/adding-committers.md

```diff
@@ -3,21 +3,61 @@ title: Adding Committers
 tags: ["pmc","committers"]
 ---
 
-The addition of committers is essential to the long-term 
-sustainability of an open source project. The PMC is responsible for
-determining who will be added as a committer.
+Identifying potential new committers, calling a vote for their recognition
+as a committer and processing the relevant documents are tasks to which
+the whole community can contribute.
 
-Please see the formal policy and process
-[documentation for adding committers](https://www.apache.org/dev/pmc.html#committer-management). 
-This page discusses best practices in how to think about new committers.
+Each project has a different approach to managing new committers. This page
+describes a common process found in many Apache projects. It also provides
+draft templates for the various communications that are necessary.
 
-In addition to the mechanisms for adding committers, give some thought
-to [practical steps you can take to attract new
-contributors](/pmc/community-growth.html).
+Some of the PMCs automatically make committers PMC members. The templates below
+have conditional `[if]` clauses for that.
+
+If the PMC has separate process for approving PMC members, see
+[new PMC member](/pmc/adding-pmc-members.html).  
+The [Contributor Ladder](https://community.apache.org/contributor-ladder.html) helps explain different roles.
 
 {{% toc %}}
 
-## Who should be a committer?
+<a name="NewCommitter-Summary"></a>
+## TL;DR - Inviting a new Committer
+
+1. Discuss the proposed committer (and optionally PMC member). (Use
+   [this template](/templates/committer-discuss.txt).)
+1. If the discussion seems to be going positively,
+   call a vote. (Use [this template](/templates/committer-vote.txt) for
```

Comment:
Suggest making link text detailed, not generic (which sounds like "click here" to me somehow).  So:

Use the [discussion template for a new committer]...